expr/interfaces/inteface.go

```diff
@@ -22,8 +22,14 @@ func (b *FunctionBase) GetEvaluator() Evaluator {
 	return b.Evaluator
 }
 
-// Evaluator is a interface for any existing expression parser
+// Evaluator is an interface for any existing expression parser.
 type Evaluator interface {
+	// Fetch populates the values map being passed into it by translating input expressions into a series of
+	// parser.MetricRequest and fetching the raw data from the configured backend.
+	//
+	// It returns a map of only the data fetched in the current invocation.
```

Comment:
Does this make sense?
```suggestion
	// It returns a map of only the data requested in the current invocation, scaled to a common step.
```